applications/EmpireApplication/tests/sdof_static_solver/results_sdof_static_ref.dat

```diff
@@ -0,0 +1 @@
+0.002533029591058445
```

Comment:
If you want to check doubles, you can do that at the python level ([see CompFlow tests](https://github.com/KratosMultiphysics/Kratos/blob/230c3a9ec26e82cc2c5674aca4db53a89594cdb4/applications/CompressiblePotentialFlowApplication/tests/potential_flow_test_factory.py#L63))

Reply:
@BasselSaridar this is not solved is it?

Reply:
You could save this value in the variable DISPLACEMENT in your sdof solver and then read it in the test.